src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/validators/OcrFormValidator.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.validators;
+
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.OcrDataField;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.out.OcrValidationResult;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.model.out.ValidationStatus;
+import uk.gov.hmcts.reform.bulkscanccdeventhandler.util.OcrFormValidationHelper;
+
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+import static org.apache.commons.lang.StringUtils.isBlank;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.model.out.ValidationStatus.ERRORS;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.model.out.ValidationStatus.SUCCESS;
+import static uk.gov.hmcts.reform.bulkscanccdeventhandler.model.out.ValidationStatus.WARNINGS;
+
+public abstract class OcrFormValidator {
```

Comment:
Is there going to be more than one subclass of it?

Reply:
I have added 2 form types at the moment. so there will be 2 subclasses.
I'm happy to change it to have 1 form type and have all validations in a single class.